iOS/dough-bros-ios/View/Add Group/AddGroupViewController.swift

```diff
@@ -83,6 +83,8 @@ class AddGroupViewController: UIViewController, UITextFieldDelegate {
         }
         
         let detailedGroupsVC = GroupDetailsViewController()
```

Comment:
force unwrap in `GroupDetailsViewController` causes app to crash, we need to call `getGroupByUID` and pass in the object before pushing to the view controller